network-api/networkapi/tests.py

```diff
@@ -465,3 +467,136 @@ def test_software_page_export_fields(self):
+            parent=pni_homepage,
+            # product fields
+            privacy_ding=True,
+            adult_content=True,
+            uses_wifi=True,
+            uses_bluetooth=True,
+            review_date=date(2025, 1, 1),
+            company='Percy Corp',
+            blurb='This is a general product specifically created for visual regression testing',
+            product_url='http://example.com/general-percy',
+            price=999,
+            worst_case='Visual regression fails',
+            cloudinary_image=cloudinary_image,
+            # general product fields
+            camera_app='Yes',
+            camera_device='No',
+            microphone_app='NA',
+            microphone_device='CD',
+            location_app='Yes',
+            location_device='No',
+            personal_data_collected='Is personal data getting collected?',
+            biometric_data_collected='Is biometric data getting collected?',
+            social_data_collected='Is social data getting collected?',
+            how_can_you_control_your_data='So, how can you control your data?',
+            data_control_policy_is_bad=True,
+            company_track_record='Needs Improvement',
+            track_record_is_bad=True,
+            track_record_details='What kind of track record are we talking about?',
+            offline_capable='Yes',
+            offline_use_description='Although it is unclear how offline capabilities work',
+            uses_ai='NA',
+            ai_uses_personal_data='Yes',
+            ai_is_transparent='No',
+            ai_helptext='The AI is a black box and no one knows how it works',
+            email='test@example.org',
+            live_chat='http://example.org/chat',
+            phone_number='1-555-555-5555',
+            twitter='@TwitterHandle',
+        )
+        self.software_product_page = SoftwareProductPageFactory.create(
+            # page fields
+            title='Software Percy Product',
+            first_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+            last_published_at=datetime(2025, 1, 1, tzinfo=timezone.utc),
+            parent=pni_homepage,
+            cloudinary_image=cloudinary_image,
+            # product fields
+            privacy_ding=True,
+            adult_content=True,
+            uses_wifi=True,
+            uses_bluetooth=True,
+            review_date=date(2025, 1, 1),
+            company='Percy Corp',
+            blurb='This is a general product specifically created for visual regression testing',
+            product_url='http://example.com/general-percy',
+            price=999,
+            worst_case='Visual regression fails',
+            # software product fields
+            handles_recordings_how='Sample software recording description',
+            recording_alert='Yes',
+        )
+
+    def test_fields_remain_untouched_and_wagtail_image_is_changed(self):
+        # Test fields before the command is run
```

Comment:
let's `from django.conf import settings` and then make this test log a string along the lines of `not running test_fields_remain_untouched_and_wagtail_image_is_changed: USE_CLOUDINARY is not set` and then exit the function if `settings.USE_CLOUDINARY` isn't `True`.